src/components/Header/Header.mdx

```diff
@@ -0,0 +1,16 @@
+import { ArgTypes, Canvas, Description, Meta, Source } from "@storybook/blocks";
+
+import * as HeaderStories from "./Header.stories.tsx";
+
+<Meta of={HeaderStories} />
+
+# Header
+
+## Overview
+
+Note: This is a temporary addition to Reservoir and should not be used in
+production unless told otherwise. Continue using the Header from
+`ds-header.nypl.org` and follow implementation instructions from the
+[nypl-header-app](https://github.com/NYPL/nypl-header-app) repository.
```

Comment:
You could also refer to the `Header` docs in Storybook.

https://nypl.github.io/nypl-design-system/reservoir/v3/?path=/docs/development-guide-header--docs